server.js

```diff
@@ -30,13 +30,13 @@ app.all('/profile-edit/server/whichrt', (req, res) => {
   const reqUri = req.query.uri
 
   if (reqUri !== null) {
-    // console.debug(`DEBUG: got server/whichrt for ${reqUri}.`)
+    // Console.debug(`DEBUG: got server/whichrt for ${reqUri}.`)
```

Comment:
We should just delete this line.

Reply:
Yeah for sure, i'm continuing the clean up ;-)